tools/external_ic.F90

```diff
@@ -659,18 +659,25 @@ subroutine get_nggps_ic (Atm, fv_domain)
     graupel = get_tracer_index(MODEL_ATMOS, 'graupel')
     ntclamt = get_tracer_index(MODEL_ATMOS, 'cld_amt')
     if (trim(source) == source_fv3gfs) then
+!--- The code here assumes:
+!       (a) that hydrostatic pressure (proportional to mass) is based on moist mass
+!           ie. dry air + water vapor
+!       (b) that tracers are tracer mass divided by total mass
+!           ie. moist mass + all condensates
+!    To the best of our understanding this is the convention in the GFS v15 and
+!    later analyses.
     do k=1,npz
       do j=js,je
         do i=is,ie
           wt = Atm%delp(i,j,k)
           if ( Atm%flagstruct%nwat == 6 ) then
-            qt = wt/(1. - (Atm%q(i,j,k,liq_wat) + &
-            Atm%q(i,j,k,ice_wat) + &
-            Atm%q(i,j,k,rainwat) + &
-            Atm%q(i,j,k,snowwat) + &
-            Atm%q(i,j,k,graupel)))
+            qt = wt/(1. + (Atm%q(i,j,k,liq_wat) + &
```

Comment:
The original code was correct, see attachment. I may have gotten confused as to which version this was. 
![image](https://user-images.githubusercontent.com/53020884/113745616-6e6f8400-96d3-11eb-849c-fdbdb710ef48.png)

Reply:
@lharris4 thanks for the review.  I guess we need to reopen this with EMC.  The dev/emc sync has the logic in external_ic::get_nggps_ic as it appears in this PR since that seemed to be the conclusion I thought we arrived at.